exonum-java-binding-core/rust/ejb-app/ejb_app_services.toml

```diff
@@ -0,0 +1 @@
+services = ["configuration"]
```

Comment:
Why this file is checked in?

Reply:
Because configuration service should be enabled by default. I can remove this file though

Reply:
But if that's the case it shall work reliably with empty or no file?